### DISCUSSION TITLE
PV Speicher per SHM steuern

### DISCUSSION THREAD

**Discussion by johanns90** [camueller/SmartApplianceEnabler#256] [Allgemein]
Hallo,

ist es möglich über SAE einen PV Speicher anzubinden? im konkreten Fall versuche ich einen Victron Multiplus zu steuern. Anforderungen sind dieselben wie bei einer Wallbox: nur mit Überschuss laden und idealerweise dann wenn dynamische Wirkleistungsbegrenzung droht.

Grüße, johanns90

**Comment by camueller** [comment]
Wenn Dein Ladecontroller PWM unterstützt, kann Du ihn vom SAE mittels [PWM-Schalter](https://github.com/camueller/SmartApplianceEnabler/blob/2.0/doc/PwmSwitch_DE.md) steuern.
Andernfalls muss Du irgend ihn irgendwie mit dem [Stufenschalter](https://github.com/camueller/SmartApplianceEnabler/blob/2.0/doc/LevelSwitch_DE.md) steuern.
